tests/client/amo/components/TestScreenShots.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { renderIntoDocument } from 'react-addons-test-utils';
+import { PhotoSwipeGallery } from 'react-photoswipe';
+
+import ScreenShots from 'amo/components/ScreenShots';
+import { shallowRender } from 'tests/client/helpers';
+
+describe('<ScreenShots />', () => {
+  const onePxImg = 'data:image/gif;base64,R0lGODlhAQABAIAAAAUEBAAAACwAAAAAAQABAAACAkQBADs=';
```

Comment:
Unless you are just trying to get it under 80 chars... in which case I guess it's okay but I really dislike short variable names like this. 😃